app/styles.css

```
@@ -272,25 +272,32 @@
   [role="option"],
   [tabindex]:not([tabindex="-1"])
 ):focus-visible {
-  --tw-ring-offset-shadow: 0 0 #0000;
-  --tw-ring-shadow: 0 0 #0000;
-  outline: 1px solid hsl(var(--ring));
-  outline-offset: 1px;
+  --tw-ring-offset-shadow: 0 0 #0000 !important;
+  --tw-ring-shadow: 0 0 #0000 !important;
+  outline: 1px solid hsl(var(--focus-border)) !important;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)